.gitignore

```diff
@@ -1,9 +1,5 @@
 node_modules
 
-/server-cert.pem
-
-/server-key.pem
-
```

Comment:
Would we ever commit these to a repo? I think we still want these ignored don't we?